final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +77,73 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+addressScary = {
+    'number' => 4351,
+    'street' => "Murphy street",
+    # no city? ok
+    'state' => "Montana",
+    'zip' => 59701
+  }
+
+scary_monster = {
+  'name' => "Frank",
+  'smell' => "Horrid",
+  'weight' => 253,
+  'citiesDestroyed' => ["Manhatten", "Hershey", "Miami"],
+  'luckyNumbers' => [3, 7, 1024, 311],
+  'address' => addressScary
+}
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+
+class SuperHero
+  attr_accessor :name, :super_power, :age
+  def initialize(name, super_power, age)
+    @name          = name
+    @super_power   = super_power
+    @age           = age
+    @@arch_nemesis = "The Syntax Error"
+    @@power_level  = 100
+    @@energy_level = 50
+    @@buff         = 40
+  end
 
 # - Create the following class methods
 #   - say_name, should print the hero's name to the terminal
 #   - maximize_energy, should update the energy_level to 1000
 #   - gain_power, should take an argument of a number and INCREASE the power_level by that number
+  def say_name(name)
+    puts "#{name}"
+  end
```

Comment:
Though the directions are below, these methods should live within your class above!
Also, for now, I would stick to a single @. Admittedly, javascript is my main programming language, so I do not know the ins and outs of when the double @@ should be used, however, I think a single '@' is most appropriate here, and I'd be sure to understand the specific use case of the double @@.